README.md

```diff
@@ -302,20 +302,18 @@ NODE_ENV=production NODE_CONFIG_ENV=prod yarn build
 NODE_ENV=production NODE_CONFIG_ENV=prod yarn start
 ```
 
-**Note: To run the app locally in production mode you'll need to create a config file for local production builds.** It must be saved as `config/local-prod-amo.js` and should look like:
+#### Running builds locally
 
-```js
-const amoCDN = 'http://127.0.0.1:4000';
+**To run the app locally in production mode you'll need to create a config file for local production builds.** Production builds can be built for different environments: `dev`, `stage` and `prod` (using the `NODE_CONFIG_ENV` env var) but only one config file is needed for these environments.
 
-module.exports = {
-  // CDN URL points to the Node server.
-  amoCDN,
+The config file must be saved as `config/local.js` and should look like:
```

Comment:
I wonder if the file shouldn't be checked in and named _local.js or something, so the instruction can be just to rename it? (that way we'd be more likely to notice it if we change something about the structure of the config files)

Reply:
Good idea.